### ISSUE TITLE
Configuration Questions

### ISSUE THREAD

**Issue by BigSnicker** [Green-Party-of-Canada-Members/gpc-decidim#183]
Highest priority at the top..

1. Is it possible to use CSS to hide ONLY the "New Proposal" button, while keeping the "New Collaborative Draft" button visible?

![image](https://github.com/Green-Party-of-Canada-Members/gpc-decidim/assets/18507363/050d70fa-0366-4a69-a1d1-8f3b60184519)

2. Can we bring back the Floating Helper box and (a) selectively hide it using CSS and (b) change the text on a per component basis using Text Customizer?

3. (Least important) Is there a way to allow multiple survey responses per participant?

**Comment by microstudi** [comment]
Hi @BigSnicker:

1. Use this CSS code (then restrict it wherever you want it):
  ```css
  .title-action__action[href$="proposals/new"] {
    display: none;
  }
  ```
2. Yes, just add the text into the text "help sections": https://wedecide.green.ca/admin/help_sections . No need for term customizer
3. As it is right now, the only way is to set the survey to allow "anonymous answers" and tell people to use the private mode of the browser. Everytime they start a new session, the can answer again the survey. If you want a more generic approach, we should hack it. It is not difficult but needs some code. Also, check this module, it might interest you: https://github.com/openpoke/decidim-module-anonymous_codes

**Comment by BigSnicker** [comment]
Great stuff... but for Question 2, on the floating helper, the text can only be set once for all modules of that type?  We were hoping to explore component level customization (and thus Term Customizer).

**Comment by paarals** [comment]
> Great stuff... but for Question 2, on the floating helper, the text can only be set once for all modules of that type? 

exactly. you can define per each module level ex: processes, assemblies, conferences.. but you can't scope a particular participatory space with that.

**Comment by BigSnicker** [comment]
Just realized that this code won't hide the button for people who can't access the URL, yes?

.title-action__action[href$="proposals/new"] {
  display: none;
}

This hides the button for admins and logged in users, but not for the browsing public.

https://wedecide.green.ca/assemblies/cmd/f/537/

![image](https://github.com/user-attachments/assets/4542aaad-adbd-4abf-97ba-d9fda77c60e8)
